setup.py

```diff
@@ -56,6 +56,12 @@
           'schema-salad >= 2.6.20170927145003, < 3',
           'typing >= 3.5.3',
           'six >= 1.8.0',
+          'prov == 1.5.1',
+          'pathlib == 1.0.1',
+          'graphviz == 0.8.2',
```

Comment:
Is graphviz 0.8.1 sufficient? TravisCI in Python 3.3 mode can't see to find 0.8.2 https://travis-ci.org/common-workflow-language/cwltool/jobs/346298771#L521

Reply:
Like other modules, I tried updating this one too to use 0.8.1 instead but the workflow would not run if the version is changed. I am unsure about the underlying reason but can explore further.

Reply:
Also tried with 0.8 but same results.